mmpose/datasets/datasets/base/base_coco_style_dataset.py

```diff
@@ -63,6 +63,7 @@ class BaseCocoStyleDataset(BaseDataset):
     def __init__(self,
                  ann_file: str = '',
                  bbox_file: Optional[str] = None,
+                 headbox_file: Optional[str] = None,
```

Comment:
If so far `headbox_file` is only for the mpii dataset, I suggest only adding it in ``MpiiDataset``